### PR TITLE
Mesh 2055/fees and runtime

### PR DESCRIPTION
**Pull request by HenriqueNogara**
Removes the fees and adds a new runtime.

### CODE REVIEW

File: pallets/runtime/private/src/constants.rs [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+    #[cfg(feature = "ci-runtime")]
+    pub const MILLISECS_PER_BLOCK: Moment = 500;
+    #[cfg(not(feature = "ci-runtime"))]
+    pub const MILLISECS_PER_BLOCK: Moment = 6000;
+    pub const SLOT_DURATION: Moment = MILLISECS_PER_BLOCK;
+    #[cfg(feature = "ci-runtime")]
+    pub const EPOCH_DURATION_IN_BLOCKS: BlockNumber = MINUTES;
+    #[cfg(not(feature = "ci-runtime"))]
+    pub const EPOCH_DURATION_IN_BLOCKS: BlockNumber = 30 * MINUTES;
```
Comment (Neopallium) [inline pallets/runtime/private/src/constants.rs]: We don't need the `ci-runtime` feature flag in this runtime.

File: pallets/runtime/private/src/runtime.rs [hunk 2]
```diff
@@ -0,0 +1,423 @@
...
+        Contracts: pallet_contracts::{Pallet, Call, Storage, Event<T>},
+        PolymeshContracts: polymesh_contracts::{Pallet, Call, Storage, Event, Config},
+
+        // Preimage register.  Used by `pallet_scheduler`.
+        Preimage: pallet_preimage::{Pallet, Call, Storage, Event<T>},
+
+        Nft: pallet_nft::{Pallet, Call, Storage, Event},
+
+        TestUtils: pallet_test_utils::{Pallet, Call, Storage, Event<T> } = 50,
+
```
Comment (Neopallium) [inline pallets/runtime/private/src/runtime.rs]: We don't want this pallet in this runtime.

Comment (Neopallium) [inline pallets/runtime/private/src/runtime.rs]: We don't want this pallet in this runtime.

File: pallets/runtime/private/src/runtime.rs [hunk 3]
```diff
@@ -0,0 +1,423 @@
...
+pub use pallet_balances::Call as BalancesCall;
+pub use pallet_staking::StakerStatus;
+pub use pallet_timestamp::Call as TimestampCall;
+
+/// 100% goes to the block author.
+pub type DealWithFees = Author<Runtime>;
+
+pub type CddHandler = polymesh_runtime_common::fee_details::DevCddHandler<Runtime>;
+
+impl<'a> TryFrom<&'a RuntimeCall> for &'a pallet_test_utils::Call<Runtime> {
```
Comment (Neopallium) [inline pallets/runtime/private/src/runtime.rs]: All references to `pallet_test_utils` should be removed.

File: pallets/runtime/private/src/runtime.rs [hunk 4]
```diff
@@ -0,0 +1,423 @@
...
+
+use pallet_asset::checkpoint as pallet_checkpoint;
+use pallet_corporate_actions::ballot as pallet_corporate_ballot;
+use pallet_corporate_actions::distribution as pallet_capital_distribution;
+use pallet_session::historical as pallet_session_historical;
+use pallet_transaction_payment::RuntimeDispatchInfo;
+use polymesh_common_utilities::constants::currency::{DOLLARS, ONE_POLY};
+use polymesh_common_utilities::constants::ENSURED_MAX_LEN;
+use polymesh_common_utilities::protocol_fee::ProtocolOp;
+use polymesh_common_utilities::TestUtilsFn;
```
Comment (Neopallium) [inline pallets/runtime/private/src/runtime.rs]: Same here.  Shouldn't be any uses of `TestUtilsFn`.

**Review by Neopallium — COMMENTED** [review]
Minor changes, otherwise looks good.

**Review by Neopallium — APPROVED** [review]

**Review by adamdossa — APPROVED** [review]

**Pull request merged**